### ISSUE TITLE
GLUE task finetune params

### ISSUE THREAD

**Issue by LiweiPeng** [google-research/text-to-text-transfer-transformer#208]
Hi, In the T5 paper page 29, you mentioned the batch size(8) used for GLUE task finetune. Can you share what learning rate you used for this? Thanks.

"We therefore use a smaller batch size of 8 length-512 sequences during fine-tuning for each GLUE and SuperGLUE task."

**Comment by sharannarang** [comment]
Hello, We used a learning rate of 0.001 for finetuning the GLUE and SuperGLUE tasks, similar to all the other finetuning experiments in the paper.
